src/npctalk.cpp

```diff
@@ -2244,6 +2264,34 @@ void dialogue::gen_responses( const talk_topic &the_topic )
     }
 }
 
+int parse_mod( const dialogue &d, const std::string &attribute, const int factor )
+{
+    // player &u = *d.alpha;
+    npc &p = *d.beta;
+    int modifier = 0;
+    if( attribute == "ANGER" ) {
+        modifier = p.op_of_u.anger;
+    } else if( attribute == "FEAR" ) {
+        modifier = p.op_of_u.fear;
+    } else if( attribute == "TRUST" ) {
+        modifier = p.op_of_u.trust;
+    } else if( attribute == "VALUE" ) {
+        modifier = p.op_of_u.trust;
```

Comment:
Reading the docs you added above, it doesn't seem like this should be the same as `"TRUST"`.